src/deploy/osp_deployer/director.py

```diff
@@ -238,10 +239,12 @@ def install_director(self):
                   " --sm_pwd " + \
                   self.settings.subscription_manager_password + \
                   " --sm_pool " + \
-                  self.settings.subscription_manager_vm_ceph
+                  self.settings.subscription_manager_vm_ceph 
```

Comment:
Ceph vm & associated settings have been removed in 16.1 branch, you can use subscription_manager_pool_vm_rhel  instead here.

Reply:
We should probably rebase then.

Reply:
Rebased. should be good now.